Products/windowZ/profiles/default/metadata.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>1</version>
+  <version>2</version>
+  <dependencies>
+    <dependency>profile-plone.app.registry:default</dependency>
```

Comment:
This is not needed since the registry is in Plone 5  by default and may be harmful. I had a case recently (in Plone 4) where this purged the registry.